### ISSUE TITLE
Flow Vector Field Multiplier Broken

### ISSUE THREAD

**Issue by StasJ** [NCAR/VAPOR#2677]
1. Create/enable a flow renderer
2. Change the number of steps to a low amount like 5
3. Change the Vector Field Multiplier to 10 or 100 from the default of 1

Nothing will change.

**Comment by shaomeng** [comment]
Well, though this behavior is confusing, it turns out to be a desired behavior, as @clyne explains the meaning of a field multiplier

> The effect of the velocity multiplier should be as if the original data were scaled by the multiplier before they are accessed by the flow renderer.

And the flow renderer attempts to have a constant number of steps across the full domain. I.e., when the field multiplier is 10X larger, then the step size is 10X smaller, maintaining the same number of steps across the field, thus showing the same flow lines. 

Reference: #2563

**Comment by clyne** [comment]
Agreed it's confusing, but I think the behavior is correct and we should reject this but
